src/b00_breeze.modelLibrary.ko.js

```diff
@@ -91,13 +91,15 @@
 
     var stype = proto.entityType || proto.complexType;
     var es5Descriptors = {};
+    vor found = false;
```

Comment:
Obviously should be `var`, not `vor`. But you are correct about the error and I'm incorporating your fix. Thanks.